addons/website_event_track/views/event_backend.xml

```diff
@@ -331,10 +331,14 @@ Tracks define the agenda of your event. These can be a talk, a round table, a me
                     <div name="event_checkboxes">
                         <field name="show_menu" groups="base.group_no_one"/>
                         <label for="show_menu" groups="base.group_no_one"/>
-                        <field name="show_tracks"/>
-                        <label for="show_tracks"/>
-                        <field name="show_track_proposal"/>
-                        <label for="show_track_proposal"/>
+                        <field name="show_tracks"
+                               attrs="{'invisible': [('show_menu', '=', False)]}"/>
+                        <label for="show_tracks"
+                               attrs="{'invisible': [('show_menu', '=', False)]}"/>
+                        <field name="show_track_proposal"
+                               attrs="{'invisible': [('show_menu', '=', False)]}"/>
+                        <label for="show_track_proposal"
+                               attrs="{'invisible': [('show_menu', '=', False)]}"/>
```

Comment:
Add the group "group_no_one" in 'show_tracks' and 'show_track_proposal' also else when the `Dedicated menu` is marked TRUE and no technical rights are given than these checkboxes will be visible.